editor-components/content/content19/content19.tsx

```diff
@@ -2,64 +2,205 @@ import * as React from "react";
-              <h1 className={this.decorateCSS("description")}>{this.getPropValue("description")}</h1>
-            </div>
-            <div className={this.decorateCSS("youtube")}>
-            <iframe title="YouTube Video" 
-            src={this.getPropValue("video-link")}
-            className={this.decorateCSS("iframe")}
-             allowFullScreen></iframe>
-              <h3 className={this.decorateCSS("below-title")}>{this.getPropValue("below-title")}</h3>
-              <h2 className={this.decorateCSS("below-text")}>{this.getPropValue("below-text")}</h2>
+            {videoImage && (
+              <div className={this.decorateCSS("video-part")}>
+                <img alt="" src={videoImage} className={this.decorateCSS("video-images")} />
+                <div className={this.decorateCSS("play-part")}>
+                  <span className="as-play" onClick={() => {
+                    this.setComponentState("is_video_visible", true);
+                  }}>
+                    {playIcon && (
+                      <div className={this.decorateCSS("play-container")}>
+                        <div className={this.decorateCSS("outer-circle")}></div>
+                        <div className={this.decorateCSS("inner-circle")}></div>
+                        <ComposerIcon name={this.getPropValue("play-icon")} propsIcon={{
+                          className: this.decorateCSS("play-image")
+                        }}></ComposerIcon>
+                      </div>
+                    )}
+                  </span>
+                </div>
+              </div>
+            )}
+            <div className={this.decorateCSS("text-side")}>
+              {this.getProp("title-text").value && (
+              <p className={this.decorateCSS("title-text")}>{this.getPropValue("title-text")}</p>
+              )}
+              {description.value && (
+                <div className={this.decorateCSS("description-div")}>
+                  <h2 className={this.decorateCSS("description")}>{this.getPropValue("description")}</h2>
+                </div>
+              )}
+              <div className={this.decorateCSS("description-author")}>
+                {authorImage && (
+                  <div className={this.decorateCSS("author-icon")}>
+                    <ComposerIcon name={this.getPropValue("author-icon")} propsIcon={{
+                      className: this.decorateCSS("author-icon-photo")
+                    }}></ComposerIcon>
+                  </div>
+                )}
+                <div className={this.decorateCSS("author-info")}>
+                  <p className={"author-description-text"}>{this.getPropValue("author-description")}</p>
+                  <h1 className={"author--text"}>{this.getPropValue("author-name")}</h1>
+                </div>
+              </div>
+          
+              <div className={this.decorateCSS("button-container")}>
+                {(this.getProp("buttons").value as any[]).map((buttonObj, index) => {
```

Comment:
don't map arrays like this.
[check here](https://github.com/Teknodev/composer-tools/blob/main/editor-components/README.md#how-to-map-things)